### ISSUE TITLE
softwareKeyboardLayoutMode "pan" works in Expo Go and Expo Build:Android but not in EAS Build

### ISSUE THREAD

**Issue by SpiriaJWF** [expo/expo#13802]
### Summary

When I set the softwareKeyboardLayoutMode to "pan" in app.json -> android, it works well in Expo Go and when I do a normal `expo build:android`. But, when I build it with EAS Build it doesn't work. My bottom elements are pushed at the top of the keyboard.

### Managed or bare workflow? If you have `ios/` or `android/` directories in your project, the answer is bare!

managed

### What platform(s) does this occur on?

Android

### SDK Version (managed workflow only)

42.0.0

### Environment

Expo CLI 4.8.1 environment info:
    System:
      OS: Windows 10 10.0.19042
    Binaries:
      Node: 14.16.1 - C:\Program Files\nodejs\node.EXE
      Yarn: 1.22.10 - ~\AppData\Roaming\npm\yarn.CMD
      npm: 6.14.12 - C:\Program Files\nodejs\npm.CMD
    IDEs:
      Android Studio: Version  4.2.0.0 AI-202.7660.26.42.7351085
    npmPackages:
      expo: ^42.0.0 => 42.0.1
      react: 16.13.1 => 16.13.1
      react-dom: 16.13.1 => 16.13.1
      react-native: https://github.com/expo/react-native/archive/sdk-42.0.0.tar.gz => 0.63.2
      react-native-web: ~0.13.12 => 0.13.18
    Expo Workflow: managed

### Reproducible demo or steps to reproduce from a blank project

```
    <View style={generalStyle.flex}>
      <KeyboardAwareScrollView extraScrollHeight={-100} contentContainerStyle={styles.container}>
        <View style={generalStyle.flex}>
          <View style={styles.containerEmailImage}>
            <Image
              resizeMode="contain"
              style={styles.emailImage}
              source={require("../../assets/images/login/email-validation.png")}
            ></Image>
          </View>

          <View style={styles.containerInput}>
            <Text style={textStyle.p2}>{t("login.emailHelp")}</Text>
            <Text style={[textStyle.formLabel, styles.label]}>{t("login.email")}</Text>
            <InputReset
              value={email}
              onChangeText={onEmailChange}
              keyboardType="email-address"
              placeholder={t("login.emailPlaceholder")}
              errorMessage={emailError}
              inputContainerStyle={emailError ? theme.Input.inputContainerError : {}}
            />
          </View>
        </View>
      </KeyboardAwareScrollView>
      <View style={generalStyle.sideContainer}> //this part is pushed on top of keyboard in the result build, but is OK on Expo Go.
        <Button
          loading={isLoading}
          disabled={isLoading || email.length === 0 || currentUser?.hasEmail}
          onPress={onValidateEmail}
          title={t("login.validateEmail")}
        />
        <LinkText
          textProps={styles.skipStepText}
          containerStyle={styles.skipStep}
          onPress={redirectNextStep}
        >
          {t("login.skipStep")}
        </LinkText>
      </View>
    </View>
    
    const styles = StyleSheet.create({
      container: {
        flexGrow: 1,
      },
      containerInput: {
        padding: theme.Container.padding,
        paddingBottom: 0,
      },
      containerEmailImage: {
        marginTop: theme.Container.padding,
        justifyContent: "center",
      },
      emailImage: {
        alignSelf: "center",
        height: 150,
      },
      skipStep: {
        paddingBottom: 10,
        height: 70,
        alignSelf: "center",
        justifyContent: "center",
      },
      skipStepText: {
        textAlign: "center",
      },
      label: {
        marginTop: 30,
      },
    });
```

**Comment by iemu** [comment]
i found a workaround....
add this to app.json file
```
{
"expo": {
    "android":{
      "softwareKeyboardLayoutMode": "pan"
    },
    "androidStatusBar": {
      "backgroundColor": "#00000000"
    }
}
}
```

**Comment by SpiriaJWF** [comment]
The problem is fix in expo-cli@4.9.1

**Comment by carlosalmeida753** [comment]
This is still happening, in my case I am using "softwareKeyboardLayoutMode": "adjustPan" in the file app.json. 

In Expo Go it works fine in android and my elements go up when keyboard opens, after eas build and installing the apk that does not happen, so it seems that "softwareKeyboardLayoutMode": "adjustPan" is being ignored.
